plugin/metrics/prometheus/metricsstore/reader.go

```diff
@@ -287,17 +289,20 @@ func promqlDurationString(d *time.Duration) string {
 	return string(b)
 }
 
-func startSpanForQuery(ctx context.Context, metricName, query string) (opentracing.Span, context.Context) {
-	span, ctx := opentracing.StartSpanFromContext(ctx, metricName)
-	ottag.DBStatement.Set(span, query)
-	ottag.DBType.Set(span, "prometheus")
-	ottag.Component.Set(span, "promql")
+func startSpanForQuery(ctx context.Context, metricName, query string) (trace.Span, context.Context) {
```

Comment:
follow the OTEL syntax/convention here:
```suggestion
func startSpanForQuery(ctx context.Context, metricName, query string) (context.Context, trace.Span) {
```